src/genie/libs/parser/asa/show_crypto_ikev2_sa.py

```diff
@@ -0,0 +1,220 @@
+"""
+show_crypto_ikev2_sa.py
+Parser for the following command:
+    * show crypto ikev2 sa
+"""
+import re
+
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import (Schema, Any, Optional)
+
+
+class ShowCryptoIkev2SaSchema(MetaParser):
+    """Schema for
+        * show crypto ikev2 sa
+    """
+
+    schema = {
+        'sessions': {
+            Any(): {
+                'session_id': int,
+                'status': str,
+                'ike_count': int,
+                'child_sa_count': int,
+                'tunnels': {
+                    Any(): {
+                        'tunnel_id': int,
+                        'local': str,
+                        'remote': str,
+                        'status': str,
+                        'role': str,
+                        'encryption_algorithm': str,
+                        Optional('key_size'): int,
+                        'hashing_algorithm': str,
+                        'dh_group': int,
+                        'authentication_sign': str,
+                        'authentication_verify': str,
+                        'lifetime': int,
+                        'activetime': int,
+                        'child_sa': {
+                            Any(): {
+                                'local_selector': str,
+                                'remote_selector': str,
+                                'esp_in': str,
+                                'esp_out': str
+                            }
+                        }
+                    },
+                },
+            },
+        },
+    }
+
+
+class ShowCryptoIkev2Sa(ShowCryptoIkev2SaSchema):
+    '''Parser for
+        * show crypto ikev2 sa
+    '''
+
+    cli_command = 'show crypto ikev2 sa'
+
+    def cli(self, output=None):
+        if output is None:
+            out = self.device.execute(self.cli_command)
+        else:
+            out = output
+
+        parsed_dict = {}
+        dict_sessions = parsed_dict.setdefault('sessions', {})
+        child_sa_index = 0
+
+        # Session-id:1, Status:UP-ACTIVE, IKE count:1, CHILD count:1
+        p1 = re.compile(
+            r'^Session-id:(?P<session_id>[\d]),\s'
+            r'Status:(?P<status>[A-Za-z\-]{1,}),\s'
+            r'IKE\scount:(?P<ike_count>[\d]),\s'
+            r'CHILD\scount:(?P<child_count>[\d])$'
+        )
```

Comment:
Please add `+`s as to make parser robust against spacing changes. Something like below would be great
```
p1 = re.compile(
    r'^Session-id:(?P<session_id>[\d+]),\s+'
    r'Status:(?P<status>[A-Za-z\-]{1,}),\s+'
    r'IKE\scount:(?P<ike_count>[\d+]),\s+'
    r'CHILD\scount:(?P<child_count>[\d+])$'
)
```

Please also apply this to other regexes